README.md

```diff
@@ -100,7 +122,8 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+Fork- copy someones remote repository to your repository
```

Comment:
A fork copies a repository to your Github *account*. You can't really copy a repository to a repository.